org.eclipse.mylyn.releng/multipass/global_setup.yml

```diff
@@ -5,6 +5,7 @@
         remote_domain: "{{ remote_dom | default('public.mydomain.de')}}"
```

Comment:
remote_domain needs to match local_domain by default.

Bugzilla unit test failed when https://mylyn.local returned public.mydomain.de as a hostname which failed dns lookup